Add Tablet Dimensions

theck13 opened the pull request:
### Fix
Add dimensions for the **_Style_** screen so that the style list items are full-width on phones and have a maximum width on tablets.  See the screenshots below for illustration.

Phone|Tablet
-|-
<kbd><a href="https://user-images.githubusercontent.com/3827611/99316838-45348100-2822-11eb-9e6a-bd2a4b66678f.png"><img src="https://user-images.githubusercontent.com/3827611/99316838-45348100-2822-11eb-9e6a-bd2a4b66678f.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/99316864-4ebde900-2822-11eb-982f-facfc27ec1c3.png"><img src="https://user-images.githubusercontent.com/3827611/99316864-4ebde900-2822-11eb-982f-facfc27ec1c3.png"></a></kbd>
<kbd><a href="https://user-images.githubusercontent.com/3827611/99316859-4cf42580-2822-11eb-8b65-a2a642e1e593.png"><img src="https://user-images.githubusercontent.com/3827611/99316859-4cf42580-2822-11eb-8b65-a2a642e1e593.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/99316871-5087ac80-2822-11eb-8800-4c3f84231711.png"><img src="https://user-images.githubusercontent.com/3827611/99316871-5087ac80-2822-11eb-8800-4c3f84231711.png"></a></kbd>

Since a `RelativeLayout` view was added as the top-level view to all the `style_list_row_*.xml` layouts, the diff of these changes looks huge unless you use the "Hide whitespace changes" setting in GitHub.  That makes the changes go from (+370/-311) to (+71/-11).

### Test
When performing the test steps below, be sure to use a small screen device (i.e. phone) as well as large screen device (i.e. tablet) in both portrait and landscape orientation.
1. Open navigation drawer.
2. Tap **_Settings_** item in drawer.
3. Tap **_Style_** setting under **_Appearance_** section.
4. Notice **_Style_** screen as shown in screenshots above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.

theck13 committed:
Add maximum width to style list row layouts for large devices

theck13 committed:
Update style list row layouts for maximum width dimension

theck13 committed:
Merge branch 'feature/add-app-styles' into add-tablet-dimensions

nbradbury commented on Simplenote/src/main/res/values/dimens.xml:
```diff
@@ -56,6 +56,7 @@
     <dimen name="sort_switch_width">40dp</dimen>
     <dimen name="style_stroke">1dp</dimen>
     <dimen name="style_stroke_activated">2dp</dimen>
+    <dimen name="style_width">-1px</dimen>
```
I'm wondering if this could benefit from a comment stating this is faking `match_parent`?

theck13 replied:
Good idea.  I added the comment in e41b0b8d.

theck13 committed:
Add comment to dimension using -1px value to explain it fakes match parent

nbradbury commented:
@theck13 Code changes look good, and I see you already addressed my minor comment. I'll merge this once it passes design review.

theck13 committed:
Merge branch 'feature/add-app-styles' into add-tablet-dimensions

SylvesterWilmott commented:
Can we use 560dp for the container width on tablet portrait and landscape?

theck13 committed:
Update maximum width of style list row layouts for large devices to 560dp

theck13 committed:
Update preview text for style list row to avoid lone words

theck13 committed:
Merge remote-tracking branch 'origin/add-tablet-dimensions' into add-tablet-dimensions

theck13 commented:
> Can we use 560dp for the container width on tablet portrait and landscape?

I updated the width in f77a8844.  I updated the preview text in 99187aa7.  That way we don't have lone words on the second line.  See the screenshots below for illustration.

Phone|Tablet
-|-
<kbd><a href="https://user-images.githubusercontent.com/3827611/100894525-d12ce680-3479-11eb-8b9e-cc37eb7454c8.png"><img src="https://user-images.githubusercontent.com/3827611/100894525-d12ce680-3479-11eb-8b9e-cc37eb7454c8.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/100894541-d427d700-3479-11eb-9839-bc136e805980.png"><img src="https://user-images.githubusercontent.com/3827611/100894541-d427d700-3479-11eb-9839-bc136e805980.png"></a></kbd>
<kbd><a href="https://user-images.githubusercontent.com/3827611/100894559-db4ee500-3479-11eb-9c1c-a850fc380d75.png"><img src="https://user-images.githubusercontent.com/3827611/100894559-db4ee500-3479-11eb-9c1c-a850fc380d75.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/100894573-ddb13f00-3479-11eb-9266-78a594dc441d.png"><img src="https://user-images.githubusercontent.com/3827611/100894573-ddb13f00-3479-11eb-9266-78a594dc441d.png"></a></kbd>

SylvesterWilmott reviewed (approved):
Looks good! Thanks.

nbradbury reviewed (approved):
:shipit:
